config/runtime.exs

```diff
@@ -60,9 +60,17 @@ if config_env() == :prod do
   gateway_registration_token = System.fetch_env!("GATEWAY_REGISTRATION_TOKEN")
   wireguard_ipv4_network = System.get_env("WIREGUARD_IPV4_NETWORK", "10.3.2.0/24")
   wireguard_ipv4_address = System.get_env("WIREGUARD_IPV4_ADDRESS")
+
+  wireguard_ipv4_masquerade =
```

Comment:
Hmm curious why you need these in the portal? I may be missing context.

Reply:
The portal send this as part of the init config to the gateway.

Reply:
Yeah, but wouldn't these be coming from the DB? Or are you thinking this will be a like a global default env var? As a stop-gap measure for 0.8 I thought the gateway would just read these env vars from the docker compose.

Reply:
I'm using these to populate the DB for the gateway when it's first created.

This way we don't need to add the configuration to read them from the enviroment in the gateway. And later on when we add the feature to create gateway from the portal we can use these as defaults.

Reply:
Kk in that case, could you also add these to the base config at `config/config.exs`?